test/parallel/test-child-process-cwd.js

```diff
@@ -66,10 +68,27 @@ function testCwd(options, expectPidType, expectCode = 0, expectData) {
     }));
 }
 
+{
+  assert.throws(() => {
+    testCwd({
+      cwd: new URL(`http://${tmpdir.path}`),
```

Comment:
There's no guarantee that `tmpdir.path` would make a valid URL:
```suggestion
      cwd: new URL(`http://example.com/`),
```